main.py

```diff
@@ -98,6 +98,37 @@ def add_new_stream(self, *args, stream_url=None, stream_quality=cfg[CONFIG_QUALI
                 "Could not open stream: The provided URL is not supported"
             )
 
+        except streamlink.exceptions.PluginError:
+            QtWidgets.QMessageBox().warning(
+                self,
+                "Error",
+                # Wierd formatting because whitespaces are also included
+                """Could not open stream!\n
+The provided URL is supported, but could not be opened.\n
+Check for spelling mistakes!"""
```

Comment:
I think you can indent these and Python will still figure it out?
Also I don't think you need the `\n` in multiline strings unless you want two newlines?